### PR TITLE
Add Post method for Dream

### PR DESCRIPTION
**Pull request by valiknet18**

### CODE REVIEW

File: src/AppBundle/Document/Dream.php [hunk 1]
```diff
@@ -57,7 +62,8 @@ class Dream
      * @var string
      *
      * @ODM\Field(type="string")
-     * @Expose()
+     * @Expose()n
```
Comment (spolischook) [inline src/AppBundle/Document/Dream.php]: typo?

Comment (valiknet18) [inline src/AppBundle/Document/Dream.php]: yes

File: src/AppBundle/Controller/AbstractController.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use FOS\RestBundle\Controller\FOSRestController;
+
+class AbstractController extends FOSRestController
```
Comment (spolischook) [inline src/AppBundle/Controller/AbstractController.php]: abstract class

File: src/AppBundle/Controller/DreamController.php [hunk 3]
```diff
@@ -2,6 +2,10 @@
```
Comment (spolischook) [inline src/AppBundle/Controller/DreamController.php]: extend this controller from AbstractController

**Comment by spolischook** [comment]
tests are failing

**Comment by spolischook** [comment]
Merge it please

**Pull request merged**